drivers/gles3/shaders/scene.glsl

```diff
@@ -13,6 +13,7 @@ DISABLE_LIGHT_DIRECTIONAL = false
 DISABLE_LIGHT_OMNI = false
 DISABLE_LIGHT_SPOT = false
 DISABLE_FOG = false
+DISABLE_FOG_DEPTH = false
```

Comment:
I would reverse the naming of this (i.e. `USE_FOG_DEPTH`). And use it instead of ``fog_mode``. We should avoid the runtime `if` statements altogether.

That way we won't include the exponential fog code when using depth fog and the code will be much clearer